core/src/main/java/org/openstack4j/model/network/ext/LoadBalancerV2.java

```diff
@@ -0,0 +1,69 @@
+package org.openstack4j.model.network.ext;
+
+import org.openstack4j.common.Buildable;
+import org.openstack4j.model.ModelEntity;
+import org.openstack4j.model.network.ext.builder.LoadBalancerV2Builder;
+import org.openstack4j.openstack.networking.domain.ext.ListItem;
+
+import java.util.List;
+
+/**
+ * An entity used to update an lbaas v2 loadbalancer
+ * @author emjburns
+ */
+public interface LoadBalancerV2  extends ModelEntity, Buildable<LoadBalancerV2Builder> {
```

Comment:
Same answer - this is used to create a load balancer. update is for updating the load balancer object. the fields are more restricted in update.